Jenkinsfile

```diff
@@ -99,141 +153,44 @@ pipeline {
-                                kubectl create secret docker-registry coherence-k8s-operator-development-secret \
-                                   --namespace test-cop2-$BUILD_NUMBER \
-                                   --docker-server=$PULL_SECRET_SERVER \
-                                   --docker-username=$PULL_SECRET_USERNAME \
-                                   --docker-password="$PULL_SECRET_PASSWORD" \
-                                   --docker-email=$PULL_SECRET_EMAIL || true
-                                kubectl create secret docker-registry ocr-k8s-operator-development-secret \
-                                   --namespace test-cop-$BUILD_NUMBER \
-                                   --docker-server=$OCR_PULL_SECRET_SERVER \
-                                   --docker-username=$OCR_PULL_SECRET_USERNAME \
-                                   --docker-password="$OCR_PULL_SECRET_PASSWORD" \
-                                   --docker-email=$OCR_PULL_SECRET_EMAIL || true
-                                kubectl create secret docker-registry ocr-k8s-operator-development-secret \
-                                   --namespace test-cop2-$BUILD_NUMBER \
-                                   --docker-server=$OCR_PULL_SECRET_SERVER \
-                                   --docker-username=$OCR_PULL_SECRET_USERNAME \
-                                   --docker-password="$OCR_PULL_SECRET_PASSWORD" \
-                                   --docker-email=$OCR_PULL_SECRET_EMAIL || true
-                            '''
+                            string(credentialsId: 'coherence-operator-docker-password', variable: 'DOCKER_PASSWORD'),
+                            string(credentialsId: 'coherence-operator-docker-username', variable: 'DOCKER_USERNAME'),
+                            string(credentialsId: 'coherence-operator-docker-server',   variable: 'DOCKER_SERVER')]) {
                             withMaven(jdk: 'JDK 11.0.3', maven: 'Maven3.6.0', mavenSettingsConfig: 'coherence-operator-maven-settings', tempBinDir: '') {
                                 sh '''
-                                    export HELM_BINARY=`which helm`
-                                    export KUBECTL_BINARY=`which kubectl`
-                                    mvn -Dbedrock.helm=''$HELM_BINARY'' \
-                                        -Dk8s.kubectl=''$KUBECTL_BINARY'' \
-                                        -Dop.image.pull.policy=Always \
-                                        -Dci.build=$BUILD_NUMBER \
-                                        -Dk8s.image.pull.secret=coherence-k8s-operator-development-secret,ocr-k8s-operator-development-secret \
-                                        -Dk8s.create.namespace=false \
-                                        -P pushTestImage -P helm-test clean install
+                                    docker login $DOCKER_SERVER -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
```

Comment:
if this script is ever used in a public jenkins, we need to make certain that obfuscation does not expose internal oracle repo names, usernames or passwords.

Reply:
DOCKER_SERVER, USERNAME and PASSWORD are secret. So, there is obfuscation in the log fie.